app/views/internal/pages/_form.html.erb

```diff
@@ -38,6 +38,15 @@
         <%= form.check_box :is_top_level_path %>
```

Comment:
This feature doesn't work with the `top level path` option: when I turn on `WIP` and `Top level path`, visiting `/<page_name` gives me an error